src/current/v24.3/sso-sql.md

```diff
@@ -175,7 +177,7 @@ You can also view all of your cluster settings in the DB Console.
     }';
     ~~~
 
-1. Instead of setting `server.jwt_authentication.jwks` to a list of static signing keys, you can set `server.server.jwt_authentication.jwks_auto_fetch.enabled` to `true` to enable automatic fetching of signing keys for the issuers specified in `server.jwt_authentication.issuers`. Signing keys are fetched from the issuer's`https://{ domain }/.well-known/openid-configuration` endpoint.
+1. Instead of setting `server.jwt_authentication.jwks` to a list of static signing keys, you can set `server.server.jwt_authentication.jwks_auto_fetch.enabled` to `true` to enable automatic fetching of signing keys for the issuers specified in `server.jwt_authentication.issuers.configuration`. Signing keys are fetched from the issuer's`https://{ domain }/.well-known/openid-configuration` endpoint.
```

Comment:
In line with the new PR https://github.com/cockroachdb/cockroach/pull/127508, we should add more examples for this value `server.jwt_authentication.issuers.configuration`, i.e.,
```
Example valid values:

'https://accounts.google.com/'
['example.com/adfs','https://accounts.google.com']
'{ "issuer_jwks_map": { "https://accounts.google.com/": "https://www.googleapis.com/oauth2/v3/certs", "example.com/adfs": "https://example.com/adfs/discovery/keys" } }'
```
We faced the issue for a customer https://github.com/cockroachlabs/support/issues/2956 and have updated this setting to hold mapping of issuer URL to JWKS(json web key set) URI

Reply:
One correction for the array of values, in light of the support issue. It needs to be:
```
'["example.com/adfs","https://accounts.google.com"]
```